baystation12.dme

```diff
@@ -550,6 +550,7 @@
 #include "code\game\mecha\combat\marauder.dm"
 #include "code\game\mecha\combat\phazon.dm"
 #include "code\game\mecha\equipment\mecha_equipment.dm"
+#include "code\game\mecha\equipment\tracking_beacon.dm"

```

Comment:
Seperated the tracking beacons out, having them in the same file as the now-defunct console was dumb